packages/backend/classic-level-index.patch

```diff
@@ -0,0 +1,11 @@
+--- node_modules/classic-level/index.backup.js	2023-01-25 18:21:31.155350735 +0100
```

Comment:
Should this file be added? I thought you've moved this to webpack

Reply:
leftover